Source/IntegrationTests/TestFiles/LitTests/LitTest/git-issues/git-issue-3855.dfy

```diff
@@ -1,5 +1,5 @@
 // RUN: %exits-with 4 %baredafny verify --show-snippets:false --allow-axioms --allow-deprecation --use-basename-for-filename "%s" > "%t".raw
-// RUN: %sed 's/after \d+ seconds/after <redacted> seconds/' %t.raw > "%t"
+// RUN: %sed 's/after [0-9]+ seconds/after <redacted> seconds/' %t.raw > "%t"
```

Comment:
:thinking:

Reply:
The `sed` on my machine somehow doesn't recognize the `\d`.